.github/workflows/pr-acceptance-ee.yml

```diff
@@ -60,6 +60,9 @@ jobs:
     runs-on: ubuntu-latest
     needs: [go-version, license-encryption-password]
     steps:
+      - uses: actions-ecosystem/action-remove-labels@v1
```

Comment:
This shouldn't work, because we made this pipeline run with read-only permissions. Did you test this in a fork? I'm concerned it will break the pipeline after we merge.